### PR TITLE
Merge remote-tracking branch 'remotes/origin/master' into PyCSW-documentation

### PR DESCRIPTION
**Pull request by epifanio**
This PR adds some notes and documentation about the pycsw library, used to implement an OGC CSW service.

### CODE REVIEW

File: source/metadata_management_heritage.rst [hunk 1]
```diff
@@ -6,8 +6,101 @@ MET Norway
 
 Service catalog
 ^^^^^^^^^^^^^^^
+The software library used to implement a Service Catalogue is based on the PyCSW python library.
+pycsw enables users to publish a catalogue of dataset descriptions (metadata) to the web, using standards based interfaces and formats, such that it can be queried and updated by catalogue clients. pycsw is simple to setup within an Apache web server (or any WSGI server) and can access metadata records stored as XML files within a database.
+
+Metadata is 'data about data', storing such things as creation-date, author, title, area-of-interest, etc, and is usually encoded as XML files, following international standards such as ISO 19115, ISO 19139 and FGDC CSDGM.
+PyCSW is the software library used to provide a catalogue service. PyCSW implements the  `OGC Catalogue Services for the Web (CSW) standard <https://www.opengeospatial.org/standards/cat>`__. The project is certified `OGC Compliant`_, and is an OGC Reference Implementation. The CSW standard defines common interfaces to discover, browse and query metadata about spatial data, web services and related resources.
+
+Core Features
```
Comment (mortenwh) [inline source/metadata_management_heritage.rst]: Could you write out the acronyms and link to the resources as well? That could make it easier to understand for a non-expert :)

Comment (epifanio) [reply source/metadata_management_heritage.rst]: I expanded most of the acronyms and added some links to the metadata specification of the several standards mentioned.

File: source/catalog_service_API.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
+Catalog Service API
+""""""""""""""""""""
+
+The software library used to implement the Catalogue Service API is based on the `PyCSW <https://pycsw.org/>`__  python
+library.  PyCSW enables users to publish a catalogue of dataset descriptions (metadata) to the web,
```
Comment (mortenwh) [inline source/catalog_service_API.rst]: From this point until "Notes for S-ENDA development", this document is a mere copy and paste from this page: https://osgeo.github.io/OSGeoLive-doc/en/overview/pycsw_overview.html

It is enough to add a link. I will never accept copy-and-paste. If the source changes, our text is immediately out of date, so we need to stick to the DRY principle (Don't Repeat Yourself [or others]). In addition, you have issues regarding intellectual property. As it is written now, it appears like we/you are the author(s) of that text. This is clearly wrong.

What we need in the S-ENDA docs is a recipe for how to use pyCSW in our context. 

Please update, and describe how to use pyCSW with our PoC prototype.

Comment (epifanio) [reply source/catalog_service_API.rst]: this should be fixed by latest commit https://github.com/metno/S-ENDA-documentation/pull/68/commits/47c523199dae6c61bd0500a9037ee8d700066cbc

**Pull request merged**